src/SprykerEco/Zed/Optivo/OptivoConfig.php

```diff
@@ -26,24 +34,124 @@ public function getRequestTimeout()
     /**
      * @return string
      */
-    public function getTokenOperationSubscribe()
+    public function getOrderListAuthCode(): string
     {
-        return $this->get(OptivoConstants::TOKEN_OPERATION_SUBSCRIBE);
+        return $this->get(OptivoConstants::ORDER_LIST_AUTHORIZATION_CODE);
     }
 
     /**
      * @return string
      */
-    public function getTokenOperationUnsubscribe()
+    public function getOrderNewMailingId(): string
     {
-        return $this->get(OptivoConstants::TOKEN_OPERATION_UNSUBSCRIBE);
+        return $this->get(OptivoConstants::ORDER_NEW_MAILING_ID);
     }
 
     /**
-     * @return mixed
+     * @return string
+     */
+    public function getOrderPaymentIsNotReceivedMailingId(): string
+    {
+        return $this->get(OptivoConstants::ORDER_PAYMENT_IS_NOT_RECEIVED_MAILING_ID);
+    }
+
+    /**
+     * @return string
+     */
+    public function getOrderShippingConfirmationMailingId(): string
+    {
+        return $this->get(OptivoConstants::ORDER_SHIPPING_CONFIRMATION_MAILING_ID);
+    }
+
+    /**
+     * @return string
+     */
+    public function getOrderCancelledMailingId(): string
+    {
+        return $this->get(OptivoConstants::ORDER_CANCELLED_MAILING_ID);
+    }
+
+    /**
+     * @return string
+     */
+    public function getServiceFormType(): string
+    {
+        return $this->getSharedConfig()->getServiceFormType();
+    }
+
+    /**
+     * @return string
+     */
+    public function getHostYves(): string
+    {
+        return $this->get(ApplicationConstants::HOST_YVES);
+    }
+
+    /**
+     * @param string|null $mailingTypeName
+     *
+     * @return string|null
+     */
+    public function getMailingIdByMailingTypeName(?string $mailingTypeName): ?string
+    {
+        if ($mailingTypeName === null) {
+            return null;
+        }
+
+        $mailingIdList = $this->get(OptivoConstants::CONFIGURATION_DEFAULT_MAILING_ID_LIST);
+
+        if (!empty($mailingIdList[$mailingTypeName])) {
+            return $mailingIdList[$mailingTypeName];
+        }
+
+        return null;
+    }
+
+    /**
+     * @return string
+     */
+    public function getOperationTypeSendTransactionEmail(): string
+    {
+        return $this->getSharedConfig()->getOperationTypeSendTransactionEmail();
+    }
+
+    /**
+     * @return string
+     */
+    public function getCustomerListAuthCode(): string
+    {
+        return $this->get(OptivoConstants::CUSTOMER_LIST_AUTHORIZATION_CODE);
+    }
+
+    /**
+     * @return string
+     */
+    public function getCustomerNewsLetterListAuthCode(): string
```

Comment:
Would be nice to rename method to same as constant name.

Reply:
done